.github/workflows/regression-tests.yml

```diff
@@ -102,6 +111,9 @@ jobs:
     name: Playwright UI Tests
     timeout-minutes: 30
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: read
```

Comment:
Why do we need permissions for just this job and not the rest?

Reply:
I read this was required to get it working but it worked without it 🤷‍♂️ . Let me remove it in the other jobs as well.

Reply:
I removed it from everywhere.